crates/solvers/src/api/routes/solve/dto/auction.rs

```diff
@@ -214,6 +215,7 @@ mod concentrated_liquidity_pool {
             liquidity::TokenPair::new(a, b)
                 .ok_or("duplicate concentrated liquidity pool token address")?
         };
+        let bps = BigDecimal::from_f32(1_000_000.).ok_or::<Error>("invalid BigDecimal".into())?;
```

Comment:
This will either always fail or never fail so an `.unwrap()` or `.expect()` seems appropriate here.

Reply:
Also `bps` are divided into `10_000` units not `1m`. Maybe that's the reason you got such a bad trade execution in staging?